### PR TITLE
Add a good-to-go route

### PR DESCRIPTION
**Pull request by keirog**
Now we’re exposing all the apps to registry health-checks, we need this
to keep the dashing tiles green ✌️

 🐿 v2.8.0

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ if (!PRODUCTION) {
   app.use(webpackHotMiddleware(compiler));
 }
 
+app.get('/__gtg', (req, res) => res.sendStatus(200));
```
Comment (AbdullahAli) [inline server.js]: Minor suggestion, but we can destruct the `res` only here, eg:

```
app.get('/__gtg', ({res}) => res.sendStatus(200));
```

Comment (keirog) [reply server.js]: 1. Does… does that work?!
2. Omg this suggestion feature is awesome 😄

Comment (AbdullahAli) [reply server.js]: It works, but the express docs are a little 💩 

```
const express = require('express');
const app = express();
const port = 3000;

app.get('/', ({ res }) => res.send('Hello World!'));

app.listen(port, () => console.log(`Example app listening on port ${port}!`));
```

Comment (keirog) [reply server.js]: TIL! Thanks!

**Pull request merged**